Disable the lib for now.

--- a/src/Makefile.am.orig	2011-10-09 19:07:47.000000000 +0200
+++ b/src/Makefile.am	2011-10-09 19:07:52.000000000 +0200
@@ -51,9 +51,9 @@
 endif
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = urjtag.pc
+noinst_DATA = urjtag.pc
 
-lib_LTLIBRARIES = liburjtag.la
+noinst_LTLIBRARIES = liburjtag.la
 liburjtag_la_LDFLAGS = -version-info 0:0:0
 
 liburjtag_la_SOURCES =
--- a/include/urjtag/Makefile.am.orig	2011-10-09 19:09:40.000000000 +0200
+++ b/include/urjtag/Makefile.am	2011-10-09 19:09:56.000000000 +0200
@@ -23,7 +23,7 @@
 
 include $(top_srcdir)/Makefile.rules
 
-pkginclude_HEADERS = \
+noinst_HEADERS = \
 	bfin.h \
 	bitmask.h \
 	bitops.h \
@@ -58,5 +58,5 @@
 	types.h \
 	usbconn.h
 
-nodist_pkginclude_HEADERS = \
+nodist_noinst_HEADERS = \
 	urjtag.h
